### PR TITLE
[staging] Fix/Changed add request button icon

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR the i have changed the icon for the add API Request in the collection page  and added new icon according to figma

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by gc-codes — APPROVED** [review]

File: src/packages/@workspaces/features/collection-list/components/folder/Folder.svelte [hunk 1]
```diff
@@ -439,6 +444,13 @@
...
           {/if}
+
+          <div
+            class="d-flex gap-2 {!collection?.items?.length
+              ? 'ms-4'
+              : 'collection-sub-btn'}"
+          ></div>
+
           <!-- {#if showFolderAPIButtons && explorer?.source === "USER"}
             <div class="mt-2 mb-2 ms-0">
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/collection-list/components/folder/Folder.svelte:455]: not required

**Comment by punitkashyup** [comment]
Please attach the issue link

**Pull request merged**